spreadsheets/collision_rank.yaml

```diff
@@ -8,7 +8,25 @@
 - {$layer: 'water', kind: 'ocean'}
 - {$layer: 'places', kind: 'country'}
 # reserved for population_rank
-- _reserved: {count: 19}
+- {$layer: 'places', kind: 'locality', population_rank: 18}
```

Comment:
The scene file also adjusts the priority based on `country_capital` and `region_capital` boolean properties, where country is best, then region, then last general locality of that same population rank. Let's adjust this logic here, too (will increase the range past the reserved gap, so some other tests may now fail and need an update).

Reply:
Something like:

```
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 18}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 18}
- {$layer: 'places', kind: 'locality', population_rank: 18}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 17}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 17}
- {$layer: 'places', kind: 'locality', population_rank: 17}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 16}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 16}
- {$layer: 'places', kind: 'locality', population_rank: 16}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 15}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 15}
- {$layer: 'places', kind: 'locality', population_rank: 15}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 14}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 14}
- {$layer: 'places', kind: 'locality', population_rank: 14}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 13}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 13}
- {$layer: 'places', kind: 'locality', population_rank: 13}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 12}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 12}
- {$layer: 'places', kind: 'locality', population_rank: 12}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 11}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 11}
- {$layer: 'places', kind: 'locality', population_rank: 11}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 10}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 10}
- {$layer: 'places', kind: 'locality', population_rank: 10}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 9}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 9}
- {$layer: 'places', kind: 'locality', population_rank: 9}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 8}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 8}
- {$layer: 'places', kind: 'locality', population_rank: 8}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 7}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 7}
- {$layer: 'places', kind: 'locality', population_rank: 7}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 6}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 6}
- {$layer: 'places', kind: 'locality', population_rank: 6}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 5}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 5}
- {$layer: 'places', kind: 'locality', population_rank: 5}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 4}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 4}
- {$layer: 'places', kind: 'locality', population_rank: 4}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 3}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 3}
- {$layer: 'places', kind: 'locality', population_rank: 3}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 2}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 2}
- {$layer: 'places', kind: 'locality', population_rank: 2}
- {$layer: 'places', kind: 'locality', country_capital: True, population_rank: 1}
- {$layer: 'places', kind: 'locality', region_capital: True, population_rank: 1}
- {$layer: 'places', kind: 'locality', population_rank: 1}
- {$layer: 'places', kind: 'locality', country_capital: True}
- {$layer: 'places', kind: 'locality', region_capital: True}
- {$layer: 'places', kind: 'locality'}
```

Reply:
Adjust to taste for the line:

> - _reserved: {count: 1}

Reply:
Fixed in 5e683a7.